src/pages/networks/forms/NetworkDefaultACLReadOnly.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FormikProps } from "formik";
+import type { FC } from "react";
+import type { NetworkFormValues } from "./NetworkForm";
+import { Icon } from "@canonical/react-components";
+import { conjugateACLAction } from "util/helpers";
+
+const NetworkDefaultACLReadOnly: FC<{
+  formik: FormikProps<NetworkFormValues>;
+}> = ({ formik }) => {
+  const egressAction = formik.values.security_acls_default_egress;
+  const ingressAction = formik.values.security_acls_default_ingress;
+
+  return (
+    <div className="u-sv1">
+      When no ACL rule matches
+      <br />
+      <Icon name="arrow-up" className="network-default-acl-readonly" />
```

Comment:
```suggestion
      <Icon name="arrow-up" className="network-default-acl-icon" />
```
I'd reuse this on edit, wdyt?